### PR TITLE
chore: update PR template

### PR DESCRIPTION
**Pull request by joshua-goldstein**
- Draft of new PR template

### CODE REVIEW

**Comment by joshua-goldstein** [comment]
@mangalaman93

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,18 +12,20 @@
  - `chore`: all chores/tests will be excluded from the CHANGELOG
  -->
 
-## Problem
```
Comment (mangalaman93) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: I think we can make the whole template a bit smaller. We probably just need JIRA Ticket, GitHub issue, [Breaking], description.

**Review by mangalaman93 — CHANGES_REQUESTED** [review]
I think we can make the whole template a bit smaller. We probably just need JIRA Ticket, GitHub issue, [Breaking], description.

**Review by mangalaman93 — CHANGES_REQUESTED** [review]
Two questions:
1. we wanted to add a list of areas, what happened to that?
2. what do you think about something like below?

```
Description: <add description>
Fixes <GitHub Issue>
Closes <JIRA Issue>

* Add [BREAKING] if it is a breaking change
* Ensure title begins with <type<(<area>)
```

**Comment by mangalaman93** [comment]
add a comment saying that, add [BREAKING] if the change is breaking something @joshua-goldstein

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,29 +1,20 @@
 <!--
- Change Github PR Title 
-
- Your title must be in the following format: 
- - `topic(Area): Feature`
+ Title format: 
+ - `Topic(Area): Feature`
  - `Topic` must be one of `build|ci|docs|feat|fix|perf|refactor|chore|test`
 
  Sample Titles:
```
Comment (mangalaman93) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: we don't need samples now I think. These are the areas that I am thinking:
acl|audit|backup|badger|cdc|dql|export|graphql|indexing|multi-tenancy|raft|restore|upgrade

**Review by mangalaman93 — CHANGES_REQUESTED** [review]
PR looks good, just one comment.

**Review by harshil-goel — APPROVED** [review]

**Pull request merged**